samples/HttpServer_FirmwareUpload/app/application.cpp

```diff
@@ -37,44 +147,35 @@ void onFile(HttpRequest& request, HttpResponse& response)
 
 int onUpload(HttpServerConnection& connection, HttpRequest& request, HttpResponse& response)
 {
+	Serial.printf("onUpload\n");
```

Comment:
If you want to use debug code don't use `Serial.printf` but `debug_d` instead.

Reply:
removed (also leftover from debugging)